README.md

```diff
@@ -38,26 +43,29 @@ Samples using the Steeltoe Connectors for connecting to backing services. Steelt
 | [SqlServerEFCore](Connectors/src/SqlServerEFCore) | [![Build Status](https://dev.azure.com/SteeltoeOSS/Steeltoe/_apis/build/status/Samples/Steeltoe-Samples-Connectors-SqlServerEFCore?branchName=main)](https://dev.azure.com/SteeltoeOSS/Steeltoe/_build/latest?definitionId=71&branchName=main) | [![Build Status](https://dev.azure.com/SteeltoeOSS/Steeltoe/_apis/build/status/Samples/Steeltoe-Samples-Connectors-SqlServerEFCore?branchName=latest)](https://dev.azure.com/SteeltoeOSS/Steeltoe/_build/latest?definitionId=71&branchName=latest) |
 
 ## [Security](Security)
+
 Samples using the Steeltoe Security packages for Authentication and Authorization with Cloud Foundry auth services, using a Redis cache for DataProtection KeyRing storage and for interacting with CredHub.
 
 | Sample | main | latest |
 | --- | --- | --- |
 | [CloudFoundrySingleSignon](Security/src/CloudFoundrySingleSignon) | [![Build Status](https://dev.azure.com/SteeltoeOSS/Steeltoe/_apis/build/status/Samples/SteeltoeOSS.Samples%20%5BSecurity_CloudFoundrySingleSignon%5D?branchName=main)](https://dev.azure.com/SteeltoeOSS/Steeltoe/_build/latest?definitionId=24&branchName=main) | [![Build Status](https://dev.azure.com/SteeltoeOSS/Steeltoe/_apis/build/status/Samples/SteeltoeOSS.Samples%20%5BSecurity_CloudFoundrySingleSignon%5D?branchName=latest)](https://dev.azure.com/SteeltoeOSS/Steeltoe/_build/latest?definitionId=24&branchName=latest) |
 
 ## [MusicStore](MusicStore)
+
 A sample that uses all of the Steeltoe components together in a microservices-based ASP.NET Core application. Adapted from the ASP.NET Core reference app: [MusicStore by Microsoft](https://github.com/aspnet/AspNetCore/tree/main/src/MusicStore).
 
 ## [FreddysBBQ](FreddysBBQ)
+
 A polyglot (Java and .NET) microservices-based sample demonstrating interoperability between Java and .NET on CloudFoundry, secured with OAuth2 Security Services and using Spring Cloud Services.
 
-# Branches
+## Branches
 
 All new development is done on the latest branch. Samples for the stable version of Steeltoe can be found on the main branch.
 
-# Documentation
+## Documentation
 
 If you are looking for documentation on how to use the Steeltoe components, you can find that [here](https://steeltoe.io/docs/).
 
-# Building & Running
+## Building & Running
```

Comment:
Moving these to the same depth as the list of samples doesn't look right to me, why was it changed?

Reply:
The change wasn't exactly intended to put them at the same level as the samples... I wanted to avoid having [multiple top-level/h1 headers](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md025.md)

I don't currently have a strong opinion of how to do this better though, so maybe just leaving it alone is best for now